ui/src/pages/Dashboard/NodeInfo/NodeAppHash.vue

```diff
@@ -185,8 +169,19 @@ export default {
       return this.pagination.perPage * (this.pagination.currentPage - 1);
     },
     total() {
-      this.paginationTotal(this.tableData.length);
-      return this.tableData.length;
+      let result;
+      if (this.searchQuery !== '') {
+        const temp = [];
+        result = this.fuseSearch.search(`=${this.searchQuery}`);
```

Comment:
why using fuzzing for searching? Is there some scenario we need fuzzy search instead of an exact search/filter?

Reply:
Hi @TheTrunk that is an exact search it so happen that is the code for the API used

Reply:
Yes but why we need some package for that?

Reply:
Hi Trunk this is a default api by our theme I did not import or use other api on search actually